### ISSUE TITLE
website issues

### ISSUE THREAD

**Issue by jeremiecoullon** [TuringLang/Turing.jl#1724]
Hello!

Just pointing out 2 quick issues about the turing website:
- It's regularly offline (though this may be only for me for some reason), ie: for a few hours at a time.
- The search bar (firefox and chrome on laptop) only shows a portion of the search results, so it's not possible to use (see screenshot below)
<img width="550" alt="turing_search_bar" src="https://user-images.githubusercontent.com/9270437/140918955-dde74148-1495-4504-b1cd-46b24b429024.png">

**Comment by torfjelde** [comment]
I don't think I've ever experienced these issues you describe before :confused: I'm using Chrome on Ubuntu.

Is this on a chrome/firefox without any extensions? Could it be that your extensions either make changes to the styling of the page or disables certain kinds of javascript?

**Comment by rikhuijzer** [comment]
> It's regularly offline (though this may be only for me for some reason), ie: for a few hours at a time.

I also cannot reproduce this I'm afraid. Site always works for me.

**Comment by cpfiffer** [comment]
> It's regularly offline (though this may be only for me for some reason), ie: for a few hours at a time.

I suspect this is more likely to be on your side, since we host through GitHub. If you can't get to any resources on turing.ml then it's probably not anything we can fix.


> The search bar (firefox and chrome on laptop) only shows a portion of the search results, so it's not possible to use (see screenshot below)

Also can't reproduce this -- do you know what your zoom/window size is? Might only occur for certain screen widths.

**Comment by jeremiecoullon** [comment]
About it being offline for me: I've found this odd as I've checked the turing using `https://downforeveryoneorjustme.com/` which said that it was all was fine. However my internet was working fine, so I'm not sure why it was doing this. So I thought perhaps it's something with my setup that other might also have. Next time I have this I'll also try it on my phone.

About the search bar: good suggestion on the zoom. If I zoom less than 100% (ie: past 80%) then I can see the entire search bar. The issue just appears for a 100% zoom for me (firefox
